src/lib/hooks.ts

```diff
@@ -24,12 +23,10 @@ function createTestFile(featureName = '', scenarioName = '') {
   )
 }
 
-function runTest(iteration, browser) {
+function runTest(browser) {
   let runner
 
-  testCafe('localhost', 1338 + iteration, 1339 + iteration).then(function(
```

Comment:
We don't need iteration to generate random ports. If you don't define it, testcafe will set differents ports for you (pararell option)